core/src/main/java/com/novoda/noplayer/exoplayer/NoPlayerExoPlayerCreator.java

```diff
@@ -51,6 +52,8 @@ ExoPlayerTwoImpl create(Context context, DrmSessionCreator drmSessionCreator, bo
 
             DefaultTrackSelector trackSelector = new DefaultTrackSelector();
 
+            MediaCodecSelector mediaCodecSelector = downgradeSecureDecoder ? SecurityDowngradingCodecSelector.newInstance() : MediaCodecSelector.DEFAULT;
```

Comment:
@joetimmins the boolean now lives on `PlayerBuilder` -> `NoPlayerCreator` and finally here where it gets converted to the codec selector